stock_location_lockdown/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Location Lockdown",
+    "summary": "Prevent to add stock on flagged locations",
```

Comment:
Instead of flagged location it would be better if you can replace it with something meaningful description.

Reply:
It looks quite meaningful to me knowing that there is the complete description on the DESCRIPTION.rst file.
Do you have a better suggestion ?

Reply:
I would suggest something like Prevent to add stock on locked locations.